server/database/models/applicant.js

```diff
@@ -142,7 +141,6 @@ const applicantSchema = new Schema({
   },
   applicationSubmittedDate: {
     type: Date,
-    required: true,
     default: Date.now,
```

Comment:
This shouldn't contain a default value or otherwise any applicant would finish the submit step before even start
And this also applies for applicationEndDate